### PR TITLE
Feature/pause game

### PR DESCRIPTION
**Pull request by anna-dp**

### CODE REVIEW

File: game/gui.py [hunk 1]
```diff
@@ -33,16 +35,31 @@ def __init__(self, screen):
...
+        ))
+        self.active_images.append((title_image, title_image_rect))
+
+        paused_image = pygame.image.load("assets/images/paused.png")
+        paused_image = pygame.transform.scale(paused_image, (PAUSED_WIDTH, PAUSED_HEIGHT))
+        title_image_rect = title_image.get_rect(topleft=(
+            (BOARD_WIDTH - PAUSED_WIDTH) / 2,
+            (BOARD_HEIGHT) / 4
+        ))
+        self.paused_images.append((paused_image, title_image_rect))
```
Comment (sarah-pham) [inline game/gui.py]: Can we move the code for title image and paused image to separate methods that we call in load_images?

File: game/gui.py [hunk 2]
```diff
@@ -77,8 +94,9 @@ def draw_block(self, x: int, y: int, color) -> None:
             rect=Rect(x * BLOCK_SIZE, y * BLOCK_SIZE, BLOCK_SIZE, BLOCK_SIZE)
         )
 
-    def draw_title(self):
-        self.screen.blit(self.graphics['title'], (TITLE_X, TITLE_Y));
+    def draw_game_images(self):
+        for img in self.active_images:
+            self.screen.blit(img[0], img[1])
```
Comment (sarah-pham) [inline game/gui.py:117]: what's img[0] and img[1]? can you add some comments or documentation

File: config.py [hunk 3]
```diff
@@ -34,5 +34,11 @@
...
-DROP_INTERVAL = 0.2 # Time taken in seconds for tetrimino to drop down by 1 position
\ No newline at end of file
+DROP_INTERVAL = 0.2 # Time taken in seconds for tetrimino to drop down by 1 position
+
+# Graphics
+TITLE_HEIGHT = 50
+TITLE_WIDTH = 330
+PAUSED_ASPECT_RATIO = 6.5
+PAUSED_HEIGHT = 30
+PAUSED_WIDTH = PAUSED_HEIGHT * PAUSED_ASPECT_RATIO
```
Comment (sarah-pham) [inline config.py]: `PAUSE_IMAGE_WIDTH`? and height

File: game/gui.py (hunk 1 above)
Comment (anna-dp) [reply game/gui.py]: How so?

As in a function for load_title_image and one for load_paused_image?

File: game/gui.py (hunk 2 above)
Comment (anna-dp) [reply game/gui.py:117]: Yes

File: game/gui.py (hunk 1 above)
Comment (sarah-pham) [reply game/gui.py]: yep

**Pull request merged**